contents/handbook/growth/sales/refunds.md

```diff
@@ -9,17 +9,17 @@ We know things happen and sometimes you might need a refund. Here’s how we han
 ### Learning Curve
 Just got off of the startup plan/new client accidentally used us a lot.
 
-We issue refunds in this category if this is the first bill >$1 and/or they meet criteria as explained above.
+We issue refunds or credits in this category if this is the first bill >$1 and/or they meet eligibility criteria as explained below.
 
 ### Unexpected Stardom
 Side project sudden volume spike
 
-We issue refunds in this category if this is the first bill >$1 and/or usage spiked by >200% compared to their average usage over the past 3 months, and the company doesn't have any revenue, or is a hobby project.
+We issue refunds or credits in this category if this is the first bill >$1 and/or usage spiked by >200% compared to their average usage over the past three months, and the company doesn't have any revenue, or is a hobby project.
 
 ### Under Attack
 Bot spike/abusive user drove traffic which in turn increased posthog usage
 
-We flag accounts with unusual activity spikes for review, and refund the overage amount once the issue has been resolved. The refund will cover any amount exceeding the average usage of the three months preceding the spike.
+We flag accounts with unusual activity spikes for review, and refund or issue credits to cover the overage amount once the issue has been resolved. The issued amount covers any amount exceeding the average usage of the three months preceding the spike.
 
 ## Eligibility Criteria
```

Comment:
Does part of the eligibility criteria include whether they've fixed the problem and things are trending down? i.e. for 'Unexpected Stardom' if they've fixed the event which is spiking?
What criteria mean that we're ready to consider processing a refund? end of spike? applied billing limit?

Reply:
Yes we do wait for them to fix the spikes before the refund, added another bullet in this section
For first time incidents we dont need to wait for them to implement billing limits as it would slow things down/create extra back and forth. Added this as something to check on subsequent incidents